lib/auth.ts

```diff
@@ -457,18 +457,10 @@ const getAuth = function (
 	 * balena.auth.register({
 	 * 	email: 'johndoe@gmail.com',
 	 * 	password: 'secret'
-	 * }).then(function(token) {
-	 * 	console.log(token);
-	 * });
+	 * }).then(function(result) {
+	 * 	console.log(`Token: ${result}`);
+	 * })
```

Comment:
Why did we remove the second/callback-based example?
Let's revert the example changes here to keep the PR clean, since at the end we didn't change that method at all.
```suggestion
	 * }).then(function(token) {
	 * 	console.log(token);
	 * });
	 *
	 * @example
	 * balena.auth.register({
	 * 	email: 'johndoe@gmail.com',
	 * 	password: 'secret'
	 * }, function(error, token) {
	 * 	if (error) throw error;
	 * 	console.log(token);
	 * });
```
Feel free to open a separate PR though if you think this is anyway improving the examples.